ibexOverview.js

```diff
@@ -0,0 +1,140 @@
+var PORT = 60000;
+var HOST = "http://dataweb.isis.rl.ac.uk" 
+//$('.btn').button();
+
+function refresh() {
+	$.ajax({
+		url: HOST + ":" + PORT + "/",
+		dataType: 'jsonp',
+		data: {"Instrument": "all"},
+		timeout: 1000,
+		error: function(xhr, status, error){ 
+			window.alert("Error: JSON not read");
+		},
+		success: function(data){ 
+			//window.alert(JSON.stringify(data));
+			display_data(data);
+		}
+	});
+}
+
+function clearBox(elementID){
+    document.getElementById(elementID).innerHTML = "";
+};
+
+function sortObject(o) {
+    var sorted = {},
+    key, a = [];
+
+    for (key in o) {
```

Comment:
As `key` is used incorrectly in the subsequent loop, key should be used here, and only here. In which case this line should be `for (var key in o) {` and `key` doesn't need to be initialised earlier on.